manifests/init.pp

```diff
@@ -89,7 +91,7 @@
 # $ssl_port::                     Port to deploy SSL enabled Tomcat server on
 #
 class candlepin (
-
+  $manage_extra_packages = $candlepin::params::manage_extra_packages,
```

Comment:
I'm not sure I like the extra_packages because I think that it can create a complicated situation if we add extra packages and some users will want to manage one package, but not the other. @ehelms what do you think?